Solutions/Corvus.Testing.AzureFunctions/Corvus/Testing/AzureFunctions/FunctionStartupException.cs

```diff
@@ -0,0 +1,29 @@
+﻿// <copyright file="FunctionStartupException.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Corvus.Testing.AzureFunctions
+{
+    using System;
+
+    /// <inheritdoc />
+    public class FunctionStartupException : Exception
+    {
+        /// <inheritdoc cref="Exception"/>
+        public FunctionStartupException()
```

Comment:
Do we need all three of these constructors? (They'd be appropriate in an exception type that was intended for other libraries to be able to throw, but I'm pretty sure that's not the intention here. This exception type is essentially an output from this library.)

And if we do need all three, shouldn't this zero-args constructor supply a default message?

Reply:
Added in response to [this comment](https://github.com/corvus-dotnet/Corvus.SpecFlow.Extensions/pull/83#discussion_r417587605).

Reply:
Resolved by removing the ProcessStartupException.